model_compression_toolkit/core/common/quantization/quantization_params_generation/error_functions.py

```diff
@@ -143,6 +144,36 @@ def _kl_error_function(x: np.ndarray,
                                range_max=range_max)
 
 
+def _kl_error_function_wrapper(x: np.ndarray,
+                               range_min: np.ndarray,
+                               range_max: np.ndarray,
+                               n_bins: int = 2048,
+                               n_bits: int = 8) -> np.ndarray:
+    """
+    Compute the error function between a tensor to its quantized version per channel.
+    The error is computed based on the KL-divergence the distributions have.
+    Number of bins to use when computing the histogram of the float tensor is passed.
+    The threshold and number of bits that were used to quantize the tensor are needed to compute the
+    histograms boundaries and the number of quantized bins.
+
+    Args:
+        x: Float tensor.
+        range_min: array of min bound on the quantization range.
+        range_max: array of max bound on the quantization range.
+        n_bins: Number of bins for the float histogram.
+        n_bits: Number of bits the quantized tensor was quantized by.
+
+    Returns:
```

Comment:
An array containing the KL-divergence between the float and quantized histograms of the tensor for each channel.